components/Modules/modules-content.tsx

```
@@ -1,6 +1,6 @@
 "use client";
 
-import React, { Suspense, useState } from "react";
+import React, { Suspense, useState, useCallback } from "react";
```

ESLint

[error] 'useCallback' is defined but never used.

(@typescript-eslint/no-unused-vars)